custom/layouts/categories/default.ejs

```diff
@@ -0,0 +1,34 @@
+<!DOCTYPE html>
```

Comment:
what did you change in this doc? I'm just flipping back and forth but I don't see any obvious diffs

Reply:
I just committed the original and added the new to you can see the difference.

https://github.com/adhocteam/nytimes-library/pull/34/commits/bc1a181b24d0275e397e5b5ad38872432c426b73